### ISSUE TITLE
evaluate top-1 video accuracy of recognition result : invalid argument for the ground truth  ?

### ISSUE THREAD

**Issue by sebyo** [kenshohara/3D-ResNets-PyTorch#280]
I tried to calculate the top-1 accuracy using the code provided  "python -m util_scripts.eval_accuracy ~/data/kinetics.json ~/data/results/val.json --subset val -k 1 --ignore" 
isn't it the 1 argument suppose to be the ground truth aka: the HMDB_1.json for exemple !

**Comment by sebyo** [comment]
@kenshohara I am trying to use your script on my own dataset and I want to visualize the data after loading it with the dataloader can you please help me on how to do it ?
